dev-tools/packaging/packages.yml

```diff
@@ -299,6 +305,67 @@ shared:
         source: '{{ elastic_beats_dir }}/specs/connectors.spec.yml'
         mode: 0644
 
+  - &agent_docker_edot_spec
+    <<: *agent_binary_spec_no_components
+    docker_variant: 'edot-collector'
+    extra_vars:
+      dockerfile: 'Dockerfile.elastic-agent.tmpl'
+      docker_entrypoint: 'docker-entrypoint.elastic-agent.tmpl'
```

Comment:
I think we would want the entrypoint to explicitly be the otel sub-command for this image

Reply:
I see this is getting handled by an env var.